sdk/core/src/moeParser/MoEngagePayloadParser.ts

```diff
@@ -43,4 +43,16 @@ export function getUserDeletionData(payload: string): UserDeletionData {
         getMoEAccountMeta(payloadJsonObject[ACCOUNT_META]),
         payloadJsonObject[MOE_DATA][IS_USER_DELETION_SUCCESS]
     );
+}
+
+export function getUserIdentitiesData(payload: string | null): { [k: string]: string } | null {
+    if (payload === null) {
+        return null;
+    }
+    const payloadJsonObject: { [k: string]: string } = JSON.parse(payload);
+    const mappedIdentities: { [k: string]: string } = {};
+    for (let [key, value] of Object.entries(payloadJsonObject)) {
+        mappedIdentities[key] = value.toString();
```

Comment:
Check if type cast is necessary. if yes, please add a comment on why.

Reply:
removed